scripts/commands/addkeyitem.lua

```diff
@@ -11,32 +11,41 @@ cmdprops =
     parameters = "ss"
 };
 
+function error(player, msg)
+    player:PrintToPlayer(msg);
+    player:PrintToPlayer("@addkeyitem <ID> <player>");
+end;
+
 function onTrigger(player, keyId, target)
-    if (keyId == nil or keyId == 0) then
```

Comment:
So caused that the other day when I moved nil checks to fix the *other* bug.

Bad teo, bad.